cms/djangoapps/contentstore/views/item.py

```diff
@@ -98,6 +98,7 @@ def xblock_handler(request, usage_key_string):
     GET
         json: returns representation of the xblock (locator id, data, and metadata).
               if ?fields=graderType, it returns the graderType for the unit instead of the above.
+              if ?course_tree=1, it returns the a course_outline summary and XBlock summary.
```

Comment:
In fact, I'd prefer that we not add a new boolean parameter each time we think of a new format. I'd rather you have a more generic option such as ``format=course_tree``. This also seems very similar to @nasthagiri's course blocks API, so maybe that can be shared between the LMS and Studio.